tests/stat_tests/test_missing_values_autodist.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+import pandas as pd
+import numbers
+
+import pytest
+
+from probatus.stat_tests import ks, psi, DistributionStatistics, AutoDist
+from sklearn.datasets import make_classification
+
+def test_missing_values_in_autodist():
+    """Test missing values have no impact in AutoDist functionality. """
+    # Create dummy dataframe
+    X, y = make_classification(10000, 10)
```

Comment:
Let's make the sample much smaller, to improve the speed of the test. 50x5 is already sufficient.

Reply:
Used 50 and 500. 500 was needed when computing the number of warning as I got additional warning on empty bucket. I could also reduce it to 50 and use 2 buckets for the tests on the number of warnings.

Reply:
Then let's not use make_classification, but construct a list ourselves:
```
[0, 1, 2, 3,...] # with 10 samples and one missing
```

This way each bucket should have one value, and we have full control over what is going on. No randomness in the test.

Reply:
Adjusted to have a reproducible test on small scale.